### PR TITLE
fix(proptypes):Add missing prop types to tabbed header

### PR DESCRIPTION
**Pull request by anusha5695**
Fixes issue: https://github.com/netguru/sticky-parallax-header/issues/148

The contentContainerStyle, scrollRef, keyboardShouldPersistTaps, refreshControl, rememberTabScrollPosition props are added in src/predefinedComponents/DetailsHeader/TabbedHeader.js, PropTypes, and src/index.d.ts

### CODE REVIEW

**Comment by anusha5695** [comment]
Hi @pi0trpietruszewski , could you check this?

**Review by 200610313 — APPROVED** [review]

File: src/predefinedComponents/TabbedHeader/TabbedHeader.js [hunk 1]
```diff
@@ -157,12 +158,22 @@ export default class TabbedHeader extends React.Component {
...
+      contentContainerStyles,
+      refreshControl,
+      rememberTabScrollPosition,
     } = this.props;
 
     return (
       <>
         <StatusBar barStyle="light-content" backgroundColor={backgroundColor} translucent />
         <StickyParallaxHeader
+          rememberTabScrollPosition={rememberTabScrollPosition}
```
Comment (maciejBudzinskiNG) [inline src/predefinedComponents/TabbedHeader/TabbedHeader.js:172]: This prop looks strange. Are you sure that this one is valid? 🤔

File: src/predefinedComponents/TabbedHeader/TabbedHeader.js [hunk 2]
```diff
@@ -213,6 +224,11 @@ TabbedHeader.propTypes = {
   tabWrapperStyle: ViewPropTypes.style,
   tabsContainerStyle: ViewPropTypes.style,
   foregroundImage: Image.propTypes.source,
+  contentContainerStyles: ViewPropTypes.style,
+  scrollRef: oneOfType([func, shape({ current: instanceOf(ScrollView) })]),
+  keyboardShouldPersistTaps: oneOf(['never', 'always', 'handled', false, true, undefined]),
```
Comment (maciejBudzinskiNG) [inline src/predefinedComponents/TabbedHeader/TabbedHeader.js:229]: You can omit the 'undefined' option here as this prop is not required.

**Review by maciejBudzinskiNG — CHANGES_REQUESTED** [review]

**Review by maciejBudzinskiNG — CHANGES_REQUESTED** [review]
Thanks for your input here, check my comments 😉

File: src/predefinedComponents/TabbedHeader/TabbedHeader.js (hunk 1 above)
Comment (dazulu) [reply src/predefinedComponents/TabbedHeader/TabbedHeader.js:172]: Seems to already exist on `TabbedHeaderProps` as `rememberTabScrollPosition?: boolean;`

**Review by IdaszakDaniel — APPROVED** [review]

**Pull request merged**